engine/crates/operation-normalizer/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "operation-normalizer"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+homepage.workspace = true
+keywords.workspace = true
+repository.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.75"
+graphql-parser = { git = "https://github.com/graphql-rust/graphql-parser", version = "0.4.0" }
```

Comment:
Shouldn't we re-use the `engine_parser` that is used by both engine v1 & v2 currently anyway? @obmarg any opinion?

Reply:
Can it also serialize back to string pretty GraphQL?

Reply:
And if yes, can we guarantee that this output never changes to something different, changing the operation_ids as we go?